<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="NoUseForward">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="LeftBar">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="invertedAppearance">
    <bool>true</bool>
   </property>
   <property name="format">
    <string>%p%</string>
   </property>
  </widget>
  <widget class="QLabel" name="NoUseLeft">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="BackwardBar">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>150</y>
     <width>25</width>
     <height>100</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="invertedAppearance">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="ForwardBar">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>25</width>
     <height>100</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="NoUseUp">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="RightBar">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="DownBar">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>330</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="format">
    <string>%p%</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="UpBar">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="format">
    <string>%p%</string>
   </property>
  </widget>
  <widget class="QLabel" name="NoUseRight">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </widget>
  <widget class="QLabel" name="NoUseDown">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
  </widget>
  <widget class="QLabel" name="NoUseBackward">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>260</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Backward</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="XAxe">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="YAxe">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="ZAxe">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="WAxe">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="JoystickMotion">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="MotionButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect !</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X Axis</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y Axis</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Z Axis</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>W Axis</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
